README.Rmd

```diff
@@ -0,0 +1,100 @@
+---
+output: github_document
+---
+
+<!-- README.md is generated from README.Rmd. Please edit that file -->
```

Comment:
I was hoping there was a way to generate the README from an rmd, thanks!

Reply:
Does `devtools::document()` or `check` automatically run `devtools::render_readme()`?

Reply:
There should be an action to do this automatically, but I couldn't find it. I know I've seen it used elsewhere though.

Reply:
Not that I know of.

Reply:
We have a handy Makefile with some chore commands, I guess in absence of something R-built-in, we use that

Reply:
Ah yea, looks like [they noped out of doing it auto](https://github.com/r-lib/devtools/issues/2330)

Reply:
Oh in a comment there, found `usethis::use_readme_rmd()`, but it's somewhat interactive (and writes a git pre-commit hook)